app/controllers/provider_interface/offer_changes_controller.rb

```diff
@@ -72,6 +71,21 @@ def render_course_options
       render :edit_course_option
     end
 
+    def render_step_for_invalid_form
+      return render_providers if @change_offer_form.errors[:provider_id].present?
+      return render_courses if @change_offer_form.errors[:course_id].present?
+
+      render_course_options
+    end
+
+    def redirect_to_amend_new_offer
```

Comment:
Is it `confirm`ing rather than `amend`ing if the offer doesn't exist yet?

Reply:
:+1: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1667/commits/ac32a612b19cab3e790aa690e3190b0518622bd6